extensions/wikia/Wall/Wall.setup.php

```diff
@@ -224,15 +224,21 @@
 $wgGroupPermissions['vstf']['wallarchive'] = true;
 $wgGroupPermissions['helper']['wallarchive'] = true;
 $wgGroupPermissions['sysop']['wallarchive'] = true;
+$wgGroupPermissions['moderator']['wallarchive'] = true;
 
 $wgGroupPermissions['*']['wallremove'] = false;
-$wgGroupPermissions['user']['wallremove'] = true;
+$wgGroupPermissions['user']['wallremove'] = false;
```

Comment:
@jsutterfield - No, unfortunately I'm without access to any of Wikia's private repos. I've gone ahead and removed the lines, though